MAINTAINERS.md

```diff
@@ -0,0 +1,199 @@
+- [Issue Triage](#issue-triage)
+  - [Step 1: Type the Issue](#step-1-type-the-issue)
+    - [`Support` for support requests](#support-for-support-requests)
+    - [`Wontfix` for misplaced or irrelevant issues](#wontfix-for-misplaced-or-irrelevant-issues)
+    - [`Need more info` for incomplete issues](#need-more-info-for-incomplete-issues)
+    - [`Stale` for abandoned issues](#stale-for-abandoned-issues)
+    - [`Bug` for things that don’t work as intended](#bug-for-things-that-dont-work-as-intended)
+    - [Good First Issues and Help Wanted Issues](#good-first-issues-and-help-wanted-issues)
+      - [Criteria for a `Help wanted` Issue](#criteria-for-a-help-wanted-issue)
+      - [Criteria for a `Good first issue`](#criteria-for-a-good-first-issue)
+    - [Other Issue Types](#other-issue-types)
+  - [Step 2: Prioritize the Issue](#step-2-prioritize-the-issue)
+    - [`priority/critical-urgent`](#prioritycritical-urgent)
+    - [`priority/important-soon`](#priorityimportant-soon)
+    - [`priority/important-longterm`](#priorityimportant-longterm)
+    - [`priority/backlog`](#prioritybacklog)
+  - [Step 3: Assign the Work](#step-3-assign-the-work)
+  - [Step 4: Follow up](#step-4-follow-up)
+- [Release process](#release-process)
+
+
+## Issue Triage
+
+<details>
+<summary>Source</summary>
+The Issue Triage section of the guide was heavily inspired by and occasionally uses the excellent <a href="https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md">Kubernetes Contributors Guide</a>.
+</details>
+
+**Maintainers are encouraged to regularly triage incoming issues and PRs**, which means:
+- A short response to the issue author acknowledging that the issue was seen
+- Evaluating and labelling the issue according to type, priority and urgency
+- Assigning the issue/PR to a contributor or maintainer for work or review
+
+Triage can happen asynchronously and continuously, or in regularly scheduled meetings.
+
+**Benefits of triaging are:**
+- Maintaining project momentum
+- Engagement of contributors by increasing responsiveness
+- Prevents piling up of issues and work
+- Prevents issues and the project itself becoming stale
+- Provides structure and an opportunity for collaborative decision-making with regards to the project trajectory and roadmap
+
+It can be beneficial to have triage be part of already existing regular meetings, and if the volume of incoming issues requires it, have a regular, public triage meeting. If there is a regular meeting, make sure it is added here and in the [CONTRIBUTING.md](/CONTRIBUTING.md). Also consider adding it to the [NEW-CONTRIBUTORS.md](/NEW-CONTRIBUTORS.MD) and explicitly invite new and prospective contributors to listen in or participate. This will give interested people a plannable, social opportunity to meet the maintainers and gain insight into the project, with both a low barrier to entry for the new contributors and a low amount of additional effort for the maintainers.
```

Comment:
This would seem like a reasonable suggestion if we had ore maintainers, but at the moment Lara is doing most of the work, so then having a meeting seems overkill (although we do have meetings, but not specifically for this).

Also, even if it is/becomes a good suggestion, it would make more sense to me for this document to describe the current practice? So I would drop this for now.

Reply:
From the current text it's not really clear which option is current practice and which is a suggestion, though. And I think for OpenPGP.js contributors and future maintainers it would be more useful to know the actual process being followed here, rather than an aspirational document - we can have that elsewhere :) Or just refer to https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md (which is already linked) for future ideas in case the project grows significantly.